dev-java/mvel/files/mvel-2.3.2-fix-for-java-10+.patch

```diff
@@ -0,0 +1,24 @@
+From 6cede923e79a17fcbce46bbfd6f16957727e5c87 Mon Sep 17 00:00:00 2001
+From: deaddowney <spamcontrol@mac.com>
+Date: Fri, 22 Jun 2018 09:38:24 -0400
+Subject: [PATCH] Mvel not working with Java 10 (11) GH Issue #159
+
+---
+ .../org/mvel2/optimizers/impl/asm/ASMAccessorOptimizer.java   | 4 +++-
+ 1 file changed, 3 insertions(+), 1 deletion(-)
+
+diff --git a/src/main/java/org/mvel2/optimizers/impl/asm/ASMAccessorOptimizer.java b/src/main/java/org/mvel2/optimizers/impl/asm/ASMAccessorOptimizer.java
+index 48b7a020b..86ef34b32 100644
+--- a/src/main/java/org/mvel2/optimizers/impl/asm/ASMAccessorOptimizer.java
++++ b/src/main/java/org/mvel2/optimizers/impl/asm/ASMAccessorOptimizer.java
+@@ -121,7 +121,9 @@
+     } else if (javaVersion.startsWith("1.6")
+             || javaVersion.startsWith("1.7")
+             || javaVersion.startsWith("1.8")
+-            || javaVersion.startsWith("9")) {
++            || javaVersion.startsWith("9")
++            || javaVersion.startsWith("10")
++            || javaVersion.startsWith("11")) {
```

Comment:
i'd do it the way [they have it in the latest code](https://github.com/mvel/mvel/blob/master/src/main/java/org/mvel2/optimizers/impl/asm/ASMAccessorOptimizer.java#L188-L194), that's more robust, this code ignores all future versions of java

Reply:
Resolved as in 6b55375055d430d537ea183322e0d19d7b331b91